src/containers/App/App.js

```diff
@@ -205,6 +205,8 @@ export function App({
   }, [isFetchingConfig, tenantNamespace]);
 
   const logoutButton = <LogoutButton getLogoutURL={() => logoutURL} />;
+  const enableLogAutoScroll = true;
```

Comment:
no need for these here, they can just be set in the PipelineRun / TaskRun containers.

Reply:
my idea was to centralize the flag for the entire app instead of having the potential of a different behaviour in PipelineRun vs TaskRun